openedx/features/pakx_features/views.py

```diff
@@ -62,3 +67,21 @@ def courses(request):
             'section': 'in-progress'
         }
     )
+
+
+@ensure_csrf_cookie
+@login_required
+def overview_tab_view(request, course_id=None):
+    course_key = CourseKey.from_string(course_id)
+    course = get_course_with_access(request.user, 'load', course_key)
+    course_block_tree = get_course_outline_block_tree(
+        request, six.text_type(course_id), request.user, allow_start_dates_in_future=True
+    )
+    context = {
+        'course_id': course_id,
+        'username': request.user.username,
+        'user': request.user,
```

Comment:
since we are sending `user`, can't we get `username` in the template from the user?

Reply:
Yes, remove it and `course_id` as well. They are not being used in the template